integrations/instagram/src/misc/outgoing-message.ts

```diff
@@ -16,11 +16,10 @@ export async function sendMessage(
 ) {
   const messengerClient = getMessengerClient(ctx.configuration)
   const recipientId = getRecipientId(conversation)
-  await send(messengerClient, recipientId)
+  const message = await send(messengerClient, recipientId)
   await ack({
     tags: {
-      // TODO: declare in definition
-      // [idTag]: messageId,
+      [idTag]: message.messageId,
```

Comment:
question: So it seems the acknowledgement here was originally broken because we weren't passing any Instagram message ID at all? 😮

Reply:
I think that we had tons of conversations because of this but not 100% sure yet